cmd/create-channel/main.go

```diff
@@ -31,15 +33,15 @@ func main() {
 	if _, err := toml.DecodeFile(CONFIG_FILE, &participantOpts); err != nil {
 		panic(err)
 	}
-	logger, logFile := utils.CreateLogger(LOG_FILE, "alice")
-	defer logFile.Close()
```

Comment:
I think we can avoid closing the log file manually, since that should happen anyways when the test/execution halts.